### ISSUE TITLE
No 'loss.backward()' during training?

### ISSUE THREAD

**Issue by pangyupo** [Goingqs/PyramidBox#7]
I'm not that familiar with Pytorch, but it seems we need to do the back propagate after computing the loss, right?

**Comment by Goingqs** [comment]
Sorry, I have fix it

**Comment by hdjsjyl** [comment]
@Goingqs  all the Loss shows nan when I execute the new code.
